instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -118,43 +123,66 @@ def _instrumented_requests_call(
 
         exception = None
 
+        recorder = RequestsInstrumentor().metric_recorder
+
+        labels = {}
+        labels["http.method"] = method
+        labels["http.url"] = url
```

Comment:
Is it possible to infer http.scheme as well here using url?

Reply:
Yes. However, if `url` is available, it takes priority since all components of the URI can be derived from it. See https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/metrics/semantic_conventions/http-metrics.md#parameterized-labels